joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForExpressionsCreator.scala

```diff
@@ -531,10 +531,18 @@ trait AstForExpressionsCreator(implicit withSchemaValidation: ValidationMode) {
     val memberAccess = MemberAccess(node.target, ".", s"@${node.attributeName}")(
       node.span.spanStart(s"${node.target.text}.${node.attributeName}")
     )
-    val op     = Operators.assignment
+
+    val assignmentOp = node.assignmentOperator match {
```

Comment:
`AssignmentOperatorNames` under `AstCreatorHelper` is a mapping for this you can use